src/server/config/schema.js

```diff
@@ -81,6 +81,10 @@ module.exports = () => Joi.object({
   })
   .default(),
 
+  ops: Joi.object({
```

Comment:
Not sure why you pointed to the 0.14.1 version of the API @jbudz, we are currently using 8.8.1